perf_dashboard/static/js/cur_pattern.js

```diff
@@ -0,0 +1,29 @@
+window.onload = function () {
+  // p90
+  let chart_mixer_both_p90_pattern = constructDataSeries(mixer_both_p90_pattern, none_mtls_base_p90_pattern);
+  let chart_none_mtls_both_p90_pattern = constructDataSeries(none_mtls_both_p90_pattern, none_mtls_base_p90_pattern);
+  let chart_v2_sd_full_nullvm_both_p90_pattern = constructDataSeries(v2_sd_full_nullvm_both_p90_pattern, none_mtls_base_p90_pattern);
+
+  p90ModesData = [];
+  p90ModesData.push(none_mtls_base_p90_pattern);
+  p90ModesData.push(chart_mixer_both_p90_pattern);
+  p90ModesData.push(chart_none_mtls_both_p90_pattern);
+  p90ModesData.push(chart_v2_sd_full_nullvm_both_p90_pattern);
+
+  generateChart("chart_p90", p90ModesData);
```

Comment:
Couldn’t you pass an array literal here and later to generate charts?

Reply:
If a consistent order is important, then it seems it is better to pass a dictionary to generateChart where you can order .

Reply:
sorry, i'm not quite get your point

Reply:
Yeah, order matters here, array can be seen as a dict, since array index can be seen as the default key.

Reply:
Dict is fine, then, we need to use the variable_name aka: label_name as the `dict_key`, i do not think this kind of refactor is necessary for now. Since we might change these labels sometime later. After our labels is stable, I can do this refactor.